### PR TITLE
f-header@v4.12.0 - Convert minor components to modules

### PR DESCRIPTION
**Pull request by FayeCarter**
**part 1 of many - Tests will be updated in a later PR**

v4.12.0
------------------------------
*April 1, 2021*

### Changed
- Add shared `nav.scss` file.
- Convert `CountrySelectorPanel`, `Logo`, `SkipToMain` and `UserNavigationPanel` components to use css modules.
- Unit and Component tests to support changes.

## UI Review Checks
**Header with options**
image

**Highlight header with Country Selector**
image

**Mobile**
image

**Mobile menu**
image

### CODE REVIEW

File: packages/components/organisms/f-header/src/assets/scss/nav.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    width: $nav-trigger-width;
+    height: $nav-trigger-height;
+    top: -100px;
+    left: -100px;
+
+    @include media('>mid') {
+        display: none;
+    }
+
+    &:checked~.c-nav-container {
```
Comment (DamianMullins) [inline packages/components/organisms/f-header/src/assets/scss/nav.scss]: There are [some failing lint errors in the build](https://app.circleci.com/pipelines/github/justeat/fozzie-components/5940/workflows/9c9f6d49-c4d5-48c0-9f3c-88d9a076850a/jobs/7522/parallel-runs/0/steps/0-106) mostly around spacing between selectors i.e. here there should be spaces around the `~`.

File: packages/components/organisms/f-header/src/assets/scss/nav.scss [hunk 2]
```diff
@@ -0,0 +1,123 @@
+@import './common.scss';
```
Comment (FayeCarter) [inline packages/components/organisms/f-header/src/assets/scss/nav.scss]: CSS moved from the `Navigation` component.

File: packages/components/organisms/f-header/src/components/Navigation.vue [hunk 3]
```diff
@@ -575,16 +576,11 @@ $nav-text-subFont                  : $font-family-base;
 $nav-icon-color                    : $color-secondary;
 $nav-icon-color--transparent       : $white;
 $nav-icon-color--mobileWhiteBg     : $grey--darkest;
-$nav-transition-duration           : 250ms;
```
Comment (FayeCarter) [inline packages/components/organisms/f-header/src/components/Navigation.vue:578]: All removed CSS moved to `assets/nav.scss`

File: packages/components/organisms/f-header/src/components/_tests/Logo.test.js [hunk 4]
```diff
@@ -44,7 +44,8 @@ describe('Logo', () => {
         expect(logo).toBeDefined();
     });
 
-    it('should have "c-icon--onTransparentBg" class if "headerBackgroundTheme" property is "transparent"', () => {
```
Comment (FayeCarter) [inline packages/components/organisms/f-header/src/components/_tests/Logo.test.js:47]: Tests will be fixed as part of later PR

**Pull request closed without merging**